cloudberry/project/dependencies.scala

```diff
@@ -27,6 +27,8 @@ object Dependencies {
     "joda-time" % "joda-time" % "2.9.3",
     "com.typesafe.akka" %% "akka-actor" % "2.4.4",
     "mysql" % "mysql-connector-java" % "5.1.24",
+    "com.oracle" % "ojdbc8"%"12.1.0.2",
```

Comment:
We'll move this file's changes to our Readme.txt file, because without the `maven` configuration thing, normal users who don't want to use Oracle can NOT build our project anymore.